cmd/kubeconfig.go

```diff
@@ -30,7 +37,7 @@ Generated kubeconfig file uses a credential plugin to get the cluster credential
 `,
 		Args: utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#kubeconfig"),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return contextCMD.UpdateKubeconfigCMD().RunE(cmd, args)
+			return contextCMD.NewKubeconfigCMD(okClientProvider).UpdateKubeconfigCMD().RunE(cmd, args)
```

Comment:
From someone using update kubeconfig, this is really confusing. It seems that 3 different commands are involved here (context, Kubeconfig and UpdateKubeconfig).

The struct `KubeconfigCMD` is not really exporting any function, it seems everything is local. So, creating a `KubeconfigCMD` seems unnecessary to later execute always a `UpdateKubeconfigCMD` to return a cobra command. 

Wouldn't it just make sense to have the function `UpdateKubeconfigCMD` which return the cobra command without having to export an "unused" `KubeconfigCMD` command?

(With "unused" I mean the fact that we are exporting it to always just get the cobra command, `Kubeconfig` CMD is not being really used outside the package)